brainstormforce/ultimate-addons-for-gutenberg
# Spectra Rename Functionality

### Description

- Rename functionality in Spectra block.

### Screenshots

- https://d.pr/v/xJvAxY

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate Gutenberg plugin
- Drag and drop Spectra Heading/Buttons
- Go to List View, select Spectra block mention
- Click on 3 dot icon > Click on rename and rename block and save.
- You will see Spectra block rename in list view.

### Checklist:
